gatsby-ssr.js

```diff
@@ -1,6 +1,39 @@
 import React from 'react';
 
 export const onRenderBody = ({ setPostBodyComponents, setHeadComponents }) => {
+  setHeadComponents(
+    [
+      /**
+       * All fonts that are linked with a preload are getting loaded before any
+       * other resources, no matter if the used or not (even if they are available
+       * locally, they are getting loaded). We therefore only list fonts
+       * that are used in almost all places.
+       *
+       * Font-format: Some browsers are smart enough to NOT preload different font types
+       * of the same font (like woff2 and woff), but as Chrome is preloading all
+       * listed font types, we will only include the most common one woff2.
+       *
+       */
+      <link
+        key="preload-regular"
+        rel="preload"
+        as="font"
+        href="/fonts/CocoGothic.woff2"
+        type="font/woff2"
+        crossOrigin="anonymous"
+      />,
+      <link
+        key="preload-bold"
+        rel="preload"
+        as="font"
+        href="/fonts/CocoGothic-Bold.woff2"
```

Comment:
Since the fonts now live in `static/fonts/` how are they being picked up by this? I couldn't find any configuration for the `static` folder that Gatsby would be aware of. Is this a Gatsby convention?

Reply:
Yes,  that is Gatsby convention. Gatsby copies all files from `static` to `public`.

https://www.gatsbyjs.com/docs/how-to/images-and-media/static-folder/

Reply:
Nice, thanks :+1: